command/command.go

```diff
@@ -27,6 +86,19 @@ func setup(c *cli.Context) (*manager.Manager, *model.Config) {
 	return &m, config
 }
 
+func stringWithDefault(c *cli.Context, key, def string) (val string) {
+	if val = c.String(key); val == "" {
+		val = def
+	}
+	return
+}
+
+func paramsLenValidation(c *cli.Context, number int, translationKey string) {
+	if len(c.Args()) < number {
+		h.PrintError("Please provide required parameters:\n" + h.T(translationKey))
```

Comment:
Validating input parameters as:

```
		if len(c.Args()) < 1 {		
			h.PrintError("You should specify the user name")		
		}
		if len(c.Args()) < 2 {		
			h.PrintError("You should specify the user password")		
		}
```

is a pain and a bit confusing for the end user, instead calling:

```
paramsLenValidation(c, 2, "user.create.args")
```

Will print something standard like:

```
Please provide required parameters:
$ ernest user create <username> <password>
```

That's based on the translation string "user.create.args" so both documentation and error refer to the same text.